src/hooks/api/scope/useRepository.ts

```diff
@@ -21,10 +21,11 @@ type GetRepositoryResponse = {
 export const useRepositoryQueryKey = (repo: string) => ["repository", repo]
 export const useRepository = (repo: string) => {
   return useQuery<Repository, Error>(useRepositoryQueryKey(repo), async () => {
-    const { data } = await contestsAPI.get<GetRepositoryResponse>(getRepositoryBranches(repo))
+    const repoName = repo.startsWith("https://github.com/") ? repo.replace("https://github.com/", "") : repo
```

Comment:
I remembered we should also do this on the auditor signup/profile pages in the future. We had a few users trying to paste their GitHub profile URL instead of just the username.

Reply:
ah yes, good idea. I'll create a util function or something then.